project_national/housing_characteristics/Ceiling Fan.tsv

```diff
@@ -1,5 +1,3 @@
-Dependency=Vacancy Status	Option=None	Option=Standard Efficiency	Option=Standard Efficiency, No usage
```

Comment:
We should keep the added dependency on Vacancy Status

Reply:
@afontani So by keep, do you mean copy resstock's version over to resstock-estimation? Or would some tsv maker code need to be updated in resstock-estimation?

Reply:
We just need to copy the "Ceiling Fan.tsv" to resstock-estimation.

Reply:
We should copy the "Ceiling Fan.tsv" from resstock to resstock-estimation.